triton_kernels/distributed_details/mesh.py

```
@@ -0,0 +1,170 @@
+        n_expts_tot: int,
+        dtype: torch.dtype,
+        device: torch.device,
+    ) -> None:
+        if self._is_initialized:
+            return
+
+        BLOCK_N = 32
+        BLOCK_M = 32
+        n_bytes_topk = n_tokens_global * n_expts_act * 4  # topk logits (float32): pessimistic estimate
+        n_bytes_topk += n_tokens_global * n_expts_act * 2  # topk indx (int16)
+        cdiv = lambda x, y: (x + y - 1) // y
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `cdiv` as a `def`

(E731)